### PR TITLE
Header title is not provided for article mode

### PR DESCRIPTION
**Pull request by jogonzal**
Fix for https://office.visualstudio.com/OneNote/_workitems/edit/3179972

### CODE REVIEW

File: src/scripts/clipperUI/components/modeButtonSelector.tsx [hunk 1]
```diff
@@ -126,6 +126,10 @@ class ModeButtonSelectorClass extends ComponentBase<{}, ClipperStateProp> {
 		};
 	}
 
+	initiallySetFocusToBackButton(element: HTMLElement) {
```
Comment (jogonzal) [inline src/scripts/clipperUI/components/modeButtonSelector.tsx]: Will revert this change, it isn't related

**Review by mttwc — APPROVED** [review]
